src/bindings/python/CMakeLists.txt

```diff
@@ -148,6 +140,11 @@ if(LINUX)
     endif()
 endif()
 
+if(CMAKE_VERSION VERSION_LESS 3.15)
+    message(${message_mode} "Cmake version 3.15 and higher is required. Provided version ${CMAKE_VERSION}")
+    set(ENABLE_WHEEL_DEFAULT OFF)
```

Comment:
Just to make sure. Is it only a message/warning? I suppose it needs to be an error -- if someone is trying to build a wheel and it is skipped with just a message, it will be the same story as for other build dependency `cython` (as far as I remember the problem was about silently skipping it in build stage).

Reply:
Above we have:
```
if(ENABLE_WHEEL)
    set(find_package_mode REQUIRED)
    set(message_mode FATAL_ERROR)
else()
    set(find_package_mode QUIET)
    set(message_mode WARNING)
endif()
```
So, if an user really wants to build wheel and sets `ENABLE_WHEEL=ON`, it turns in error if dependencies are not satisfied

Reply:
Great, resolved instantly!